app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,384 @@
+    <string name="message_copy_txid">Transaktions-ID in Zwischenablage kopiert!</string>
+    <string name="message_noselect_seed">Auswahl des Seeds aus Sicherheitsgründen deaktiviert!</string>
+    <string name="message_noselect_key">Auswahl des Spend-Keys aus Sicherheitsgründen deaktiviert!</string>
+    <string name="message_nocopy">Kopieren aus Sicherheitsgründen deaktiviert!</string>
+
+    <string name="message_exchange_failed">Umtausch-Rate nicht verfügbar!\nNutze XMR/XMR oder versuche es noch einmal</string>
+
+    <string name="generate_title">Wallet erstellen</string>
+    <string name="generate_name_hint">Wallet Name</string>
+    <string name="generate_password_hint">Wallet Passwort</string>
+    <string name="generate_bad_passwordB">Passwortphrasen stimmen nicht überein</string>
+    <string name="generate_empty_passwordB">Passwortphrase darf nicht leer sein</string>
+    <string name="generate_buttonGenerate">Erstell mir jetzt endlich ein Wallet!</string>
+    <string name="generate_seed">mnemonischer Seed</string>
+    <string name="generate_button_accept">Ich habe den Mnemonischen Seed notiert!</string>
+
+    <string name="generate_button_backup">Backup</string>
+    <string name="generate_button_export">Schlüssel exportieren</string>
+    <string name="generate_button_delete">LÖSCHEN</string>
+
+    <string name="generate_wallet_name">Gib mir einen Namen!</string>
+    <string name="generate_wallet_exists">Wallet existiert bereits!</string>
+    <string name="generate_wallet_dot">Kann nicht mit einem . beginnen</string>
+    <string name="generate_wallet_creating">Erstelle Wallet</string>
+    <string name="generate_wallet_created">Wallet erstellt</string>
+    <string name="generate_wallet_create_failed">Walleterstellung fehlgeschlagen</string>
+
+    <string name="generate_restoreheight_error">Gib eine Nummer oder ein Datum (JJJJ-MM-TT) ein</string>
+
+    <string name="generate_wallet_type_key">Schlüssel</string>
+    <string name="generate_wallet_type_new">Neu</string>
+    <string name="generate_wallet_type_seed">Seed</string>
+    <string name="generate_wallet_type_view">View</string>
+
+    <string name="generate_address_hint">Öffentliche Adresse</string>
+    <string name="generate_viewkey_hint">View-Key</string>
+    <string name="generate_spendkey_hint">Spend-Key</string>
+    <string name="generate_mnemonic_hint">25-Wörter mnemonischer Seed</string>
+    <string name="generate_restoreheight_hint">Wiederherstellungshöhe oder Datum (JJJJ-MM-TT)</string>
+
+    <string name="generate_wallet_label">Wallet</string>
+    <string name="generate_password_label">Passwort</string>
+    <string name="generate_address_label">Öffentliche Adresse</string>
+    <string name="generate_viewkey_label">View-Key</string>
+    <string name="generate_spendkey_label">Spend-Key</string>
+    <string name="generate_mnemonic_label">mnemonischer Seed</string>
+    <string name="generate_restoreheight_label">Wiederherstellungshöhe:</string>
```

Comment:
in the help text this field is called "Restore Height"

Reply:
Possible would be "Blockchainscan ab Blockhöhe:" meaning "Scan blockchain from blockheight:"
mnemonischer must be Mnemonischer too.

Reply:
"Scan ab Blockhöhe" ok? (It's a label which should not be very long)